src/reader.jl

```diff
@@ -0,0 +1,41 @@
+"""Parses a Policy XML file and returns `alphavectors` and `alphaactions`
+This should be able to handle any policy in the `.policy` format, which includes
+  policies written by `POMDPs.jl` or `APPL`.
+
+`alphavectors` is a matrix containing the alpha vectors where each row corresponds to a
+  different alpha vector.
+
+`alphaactions` is a vector containing the list of action indices, where ach index
+  corresponds to action associated with the alpha vector row.
+These are `0-indexed`, so `0` maps to the first action.
+"""
+
+function read_pomdp(filename::String)
```

Comment:
This is an exceedingly strange name for a function that reads in alpha vectors. I think we could name it something better and deprecate `read_pomdp`.

Reply:
I was also confused by its name. 😅

Perhaps `read_policy`?

Then `read_pomdp` (at some time TBD) would go from `.pomdpx` to `POMDP`? Though I’m not sure the utility of this.